src/Network/Network/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 --->
 
 ## Upcoming Release
+* Corrected typo in `New-AzPublicIpAddress` example 3 to include $ before variable name
```

Comment:
no need to add change log for doc fix, please remove this line

Reply:
Got it, thanks!